docs/beamrunner/release/1.2.0/troubleshoot.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: docs
+title:  Troubleshooting IBM Streams Runner for Apache Beam
+navtitle: Troubleshooting
+description: Troubleshooting IBM® Streams Runner for Apache Beam can be conducted to investigate any issues that occur while launching, running, or monitoring applications.

```

Comment:
A little wordy - I'm not sure if I've moved the product name to the right place -- Either:
"You can troubleshoot any issues that occur when you launch, run, or monitor IBM® Streams Runner for Apache Beam applications."
or:
"You can troubleshoot any issues that occur when you launch, run, or monitor applications that use IBM® Streams Runner for Apache Beam."